recipes/holoviews-1.5.0/meta.yaml

```diff
@@ -0,0 +1,41 @@
+package:
+  name: holoviews
+  version: 1.5.0
+
+source:
+  path: .
```

Comment:
Actually you can use any checksum for the source tarball. You can get the `md5` from PyPI, and that is fine, but we are recommending `sha256` because it is more robust.